### ISSUE TITLE
Needs -fcommon to build

### ISSUE THREAD

**Issue by clausecker** [embyte0/nast#1]
This project has lots of duplicate symbol errors if compiled without `-fcommon`.  Please check if you can get this fixed and perhaps cut a new release.

**Comment by embyte0** [comment]
Hi @clausecker, 

Thanks for your feedback, very much appreciated. 

Saying that I developed this project back in the years, I will be very more than happy if someone would show interest in updating it to current dev standards ;-)

Cheers

**Comment by clausecker** [comment]
I'm happy that you still care about it!  We were about to throw [net/nast](https://www.freshports.org/net/nast) out of the FreeBSD ports collection when I noticed that moved it to a new upstream.  I hope your tool will help lots of people.

**Comment by embyte0** [comment]
Exactly, few months back while reworking my personal webpage (at madlab.it) I decided to bring back to life the old projects by adding them to my github repo so to foster new collaboration. Thanks for keeping it in the FreeBSD ports if possible :)
